tests/test_twisted.py

```diff
@@ -184,6 +184,19 @@ def test_catchesKeyCertificateMismatch(self, tmpdir):
 
         assert str(excinfo.value).startswith("No certificate matching ")
 
+    def test_catchesMultipleDHParams(self, tmpdir):
+        pemFile = tmpdir.join('multiple_params.pem')
+        pemFile.write(KEY_PEM + CERT_PEMS[0] + DH_PEM + DH_PEM)
+
+        with pytest.raises(ValueError) as excinfo:
+            certificateOptionsFromFiles(
+                str(pemFile)
+            )
+
+        assert str(excinfo.value) == (
```

Comment:
I prefer expected == actual because it leads to nice alignments if you have multiple (only pointing it out tho because you’ll have to touch it anyways for the docstring)